.github/workflows/convert.yml

```diff
@@ -0,0 +1,56 @@
+name: convert precommit and test
+run-name: convert precommit and test
+on:
+  pull_request:
+    paths:
+      - convert/**
+      - .github/workflows/convert.yml
+jobs:
+  convert-pre-commit-actions:
+    strategy:
+      matrix:
+        python-version: [ "3.8.15" ]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: "pip"
+      - name: Run isort
+        uses: isort/isort-action@v1.1.0
+        with:
+          sort-paths: convert/*
+          configuration:
+            --check-only
+            --profile=black
+            --line-length=79
+            --skip=tests
```

Comment:
I would also add --skip = alembic, because sometimes automatically created alembic revisions are not consistent with black linter.
Please see, let's say, .github/workflows/jobs.yaml